ghost/members-api/lib/services/member-bread.js

```diff
@@ -64,9 +64,12 @@ module.exports = class MemberBREADService {
         }
 
         const subscriptionProducts = (member.subscriptions || [])
-            .filter(sub => sub.status !== 'canceled')
+            .filter(sub => this.memberRepository.isActiveSubscriptionStatus(sub.status))
```

Comment:
This fixes that we also add a complimentary plan if all subscriptions are inactive, but we do have an active tier

Reply:
👍